core/include/drivers/semihosting_console.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2024 Andes Technology Corporation
+ */
+#ifndef __DRIVERS_SEMIHOSTING_CONSOLE_H
+#define __DRIVERS_SEMIHOSTING_CONSOLE_H
+
+#include <drivers/serial.h>
+
+struct semihosting_console_data {
```

Comment:
an inline description comment is welcome

Reply:
OK will add it

Reply:
Addressed in https://github.com/OP-TEE/optee_os/pull/6700/commits/c036e76844763d61d6abe980e9b7ac880993cd5d